occurrence-integration-tests/src/test/resources/application-test.yml

```diff
@@ -1,3 +1,4 @@
+defaultChecklistKey: d7dddbf4-2cf0-4f39-9b2a-bb099caae36c
 appkeys:
   file: /Users/marcoslopezgonzalez/dev/gdrive/WS/appkeys.properties
```

Comment:
oh I just saw my path there, my bad at some point :)

Reply:
ive replaced with the placeholder value (from earlier commits)